source/docs/gitlab-pages.md

```diff
@@ -19,26 +20,26 @@ before_script:
 
 pages:
   script:
-    - hexo generate
+    - npm run build
```

Comment:
```suggestion
    - hexo generate
```

Although `hexo-starter` has `npm run build` supports, the documentation should still use `hexo g`.

Reply:
I would remove the global install and use npx hexo g

Reply:
```suggestion
    - npx hexo g
```